### PR TITLE
downgrade quic-go

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
Fix: #33314

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -51,7 +51,7 @@ require (
 	github.com/kr/pretty v0.2.1
 	github.com/kylelemons/godebug v1.1.0
 	github.com/lestrrat-go/jwx v1.2.0
-	github.com/lucas-clemente/quic-go v0.21.0
+	github.com/lucas-clemente/quic-go v0.20.0
```
Comment (ericvn) [inline go.mod:54]: Is there a reason to not move back to 0.20.1? A quick test show the later release still works.

Comment (hzxuzhonghu) [reply go.mod:54]: https://github.com/lucas-clemente/quic-go/issues/3194

Comment (hzxuzhonghu) [reply go.mod:54]: $ go version
go version go1.16.5 windows/amd64

Comment (ericvn) [reply go.mod:54]: @hzxuzhonghu Note my comment was 0.20.1, not 0.21.0. I understand 0.21.0 has issues. I was asking why we shouldn't downgrade to the latest 0.20.x release.

Comment (hzxuzhonghu) [reply go.mod:54]: Oh, i missed v0.20.1

Comment (hzxuzhonghu) [reply go.mod]: done @ericvn  Thanks

**Review by ericvn — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

**Pull request merged**